pkgs/tools/security/wad/default.nix

```diff
@@ -0,0 +1,45 @@
+{ lib
+, buildPythonApplication
+, fetchPypi
+, mock
+, pytestCheckHook
+, six
+}:
+
+buildPythonApplication rec {
+  pname = "wad";
+  version = "0.4.6";
+
+  src = fetchPypi {
+    inherit pname;
+    inherit version;
+    sha256 = "02jq77h6g9v7n4qqq7qri6wmhggy257983dwgmpjsf4qsagkgwy8";
+  };
+
+  propagatedBuildInputs = [
+     six
```

Comment:
```suggestion
    six
```

Reply:
I guess that I've over-written `nixpkgs-fmt`'s run. Thanks.